File: benchmarks/src/main/resources/js/fibonacci.iterative.js
function fibonacci(n) {
  if (n <= 1) {
    return n;
  }

  let nMinusOne = parseInt(1);
  let nMinusTwo = parseInt("0");
  for (let index = 2; index <= n; ++index) {
    nMinusOne += nMinusTwo;
    nMinusTwo = nMinusOne - nMinusTwo;
  }

  return nMinusOne;
}

File: benchmarks/src/main/resources/ruby/fibonacci_iterative.rb
def fibonacci(n)
  if (n <= 1)
    return n
  end
  nMinusTwo = 0
  nMinusOne = 1
  for index in 2..n
    nMinusOne += nMinusTwo
    nMinusTwo = nMinusOne - nMinusTwo
  end
  return nMinusOne
end

if __FILE__ == $0
  n = ARGV[0].to_i
  puts(fibonacci(n))
end